README.md

```diff
@@ -17,7 +25,38 @@ string and see what result you get (hint: you'll know when you've done it right)
 !YTIRCO!IQIIQIDEMGMMIM FO YMJMMSM!RA !EGEEJEHT ROEOOSOF PAEJEEBEL TN!AIKIITIG ENVNNMNO ,GQGGCGN!ILEKIZIISIRT A RJRRDROF PETOTTJTS LLZLLEL!AMSXSSMS ENODOOSO
 ```
 
-## Expanding your programming comprehension
+## Problem 2 - Number Wrangling
+
+Write a `EquationGuesser` class that in its constructor takes as one argument an array of integers
+(we'll call these the operands), and as another argument another integer (we'll call this the
+desired result). Your class should provide a method `guess` that takes an integer (guesses).
+When `guess` is called, your class should attempt to output `guesses` X string representations of
+equations, their result, and the difference between that result and the desired result. Finally,
+the class should output the number of guesses and what the best guess was.
+
+e.g.
+
+```
+> equation_guesser = EquationGuesser.new([1, 2, 3, 4], 24)
+> equation_guesser.guesses(3)
+1 + 2 + 3 + 4 = 10 (-14)
+1 * 2 * 3 * 4 = 24 (0)
+Guesses: 2 Best Guess: 1 * 2 * 3 * 4 = 24 (0)
+```
+
+If the equation is exactly equal to the desired result, the program should exit. For the sake of
+this exercise, you only need consider positive integers (i.e. use integer division). Your equations
+should honour [BODMAS](https://www.mathsisfun.com/operation-order-bodmas.html) but you don't need to
```

Comment:
How about "standard operator precedence" or "the usual rules of arithmetic", with a side of "don't worry about bracketing subexpressions when generating guesses"?